### DISCUSSION TITLE
About File Server Migration

### DISCUSSION THREAD

**Discussion by NanoAgentCode** [minio/minio#15215] [Q&A] [answered]
Minio team:
        I used to use the fastDFS file server,
 and now I map the file resources of the source fastDFS directly to a bucket of MiniO through the startup command. 
        However, I encountered a problem when downloading the file. Did Minio get any meta information when uploading the file? The specified key did not exist.
        I was hoping I could get some help. Thank you to the Minio team.

**Comment by klauspost** [comment, marked as answer]
There is literally no useful information in this question. If you put in all information about your setup and what you  did maybe someone can help you.
